### PR TITLE
Update quickstart guide

### PR DESCRIPTION
**Pull request by biancabuzea200**
DEV-7936

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-7936 update quickstart guide](https://app.clickup.com/t/2645698/DEV-7936)

File: docs/guides/getting-started.md [hunk 1]
```diff
@@ -34,57 +34,60 @@ We will use the [`@erc725/erc725.js`](../tools/erc725js/getting-started) NPM pac
...
-// Here we try to load an already created key from the localstorage
-web3.eth.accounts.wallet.load(myDummyPassword);
+// Check if there is already a key, if none exists, we create a new one
+if (privateKey === '') {
+  let account = web3.eth.accounts.create();
 
-// If none exists we create a new key
-if (!web3.eth.accounts.wallet.length) {
-  web3.eth.accounts.wallet.create(1);
-  web3.eth.accounts.wallet.save(myDummyPassword);
```
Comment (Hugoo) [inline docs/guides/getting-started.md:48]: @biancabuzea200 Why did you remove this part?

Comment (biancabuzea200) [reply docs/guides/getting-started.md:48]: I tried with the old version today that uses wallet.save, wallet.load. I found that those are browser-only (https://web3js.readthedocs.io/en/v1.2.11/web3-eth-accounts.html), so the tutorial can only be run from the browser. With the .env setup, users have more flexibility over where they can run the tutorial from. Do we want to limit devs to only run in the browser?

**Review by Hugoo — APPROVED** [review]
